R/pkg/R/SQLContext.R

```diff
@@ -110,10 +110,11 @@ sparkR.conf <- function(key, defaultValue) {
     value <- if (missing(defaultValue)) {
       tryCatch(callJMethod(conf, "get", key),
               error = function(e) {
-                if (any(grep("java.util.NoSuchElementException", as.character(e)))) {
+                estr <- as.character(e)
+                if (any(grep("java.util.NoSuchElementException", estr, fixed = TRUE))) {
```

Comment:
oh this should be `grepl` (or we should to `length`)

Reply:
technically this would be a bugfix but `any` would mostly work since `any(integer())` is `FALSE`